packages/peregrine/lib/talons/Breadcrumbs/breadcrumbs.gql.js

```diff
@@ -1,5 +1,14 @@
 import { gql } from '@apollo/client';
 
+export const GET_STORE_CONFIG_DATA = gql`
+    query GetStoreConfigData {
```

Comment:
Minor nit; there is an eslint rule I'd love to have enabled right now for unique named operations, which is required to start doing some fun code and type generation. Couldn't take on that scope with the latest `validate-queries` PR, so that's still in the backlog. Consider not adding to this debt and giving new operations more unique names, eg. `GetStoreConfigForBreadcrumbs`.

Reply:
thanks @tjwiebell, I didn't know about this rule, I'll change the names